Horrible hack to make the configure phase (./setup dependencies) unattended.
--- a/setup
+++ b/setup
@@ -593,6 +593,9 @@
 
     The "answer" return value is one of "yes" or "no".
     """
+
+    return True
+
     valid = {"yes":True,   "y":True,  "ye":True, "no":False,     "n":False}
     if default == None: prompt = " [y/n] "
     elif default == "yes": prompt = " [Y/n] "
